examples/data-objects/inventory-app/src/schema.ts

```diff
@@ -3,24 +3,52 @@
  * Licensed under the MIT License.
  */
 
-import { FieldKinds, SchemaAware, SchemaBuilder, ValueSchema } from "@fluid-experimental/tree2";
+import {
+	AllowedUpdateType,
+	FieldKinds,
+	SchemaAware,
+	SchemaBuilder,
+	ValueSchema,
+} from "@fluid-experimental/tree2";
 
 const builder = new SchemaBuilder("inventory app");
-export const float64 = builder.leaf("number", ValueSchema.Number);
-export const string = builder.leaf("string", ValueSchema.String);
+// REV: I'm not sure why this is necessary...
+const float64 = builder.leaf("number", ValueSchema.Number);
+const string = builder.leaf("string", ValueSchema.String);
 
-export const part = builder.struct("Contoso:Part-1.0.0", {
+const part = builder.struct("Contoso:Part-1.0.0", {
+	// REV: ...e.g. why this isn't just SchemaBuilder.field(FieldKinds.value, ValueSchema.String)
 	name: SchemaBuilder.field(FieldKinds.value, string),
 	quantity: SchemaBuilder.field(FieldKinds.value, float64),
 });
 
-export const inventory = builder.struct("Contoso:Inventory-1.0.0", {
+// REV: Building this up as a series of builder invocations makes it hard to read the schema.
+// Would be nice if instead we could define some single big Serializable or similar that laid the
+// schema out and then pass that in.
+const inventory = builder.struct("Contoso:Inventory-1.0.0", {
 	parts: SchemaBuilder.field(FieldKinds.sequence, part),
 });
 
-export const rootField = SchemaBuilder.field(FieldKinds.value, inventory);
-export type RootField = SchemaAware.TypedField<typeof rootField>;
-
-export const schema = builder.intoDocumentSchema(rootField);
+// REV: The rootField feels extra to me.  Is there a way to omit it?  Something like
+// builder.intoDocumentSchema(inventory)
+const rootField = SchemaBuilder.field(FieldKinds.value, inventory);
+const schema = builder.intoDocumentSchema(rootField);
 
 export type Inventory = SchemaAware.TypedNode<typeof inventory>;
+
+export const schemaPolicy = {
```

Comment:
```suggestion
export const schemaPolicy: InitializeAndSchematizeConfiguration<typeof rootField> = {
```

(with an import)

Reply:
Made this change in latest.  Got me thinking about why this is merged schema + initialize though, added a REV comment for that.